### PR TITLE
show more results does not close when clicked 2nd time

### PR DESCRIPTION
**Pull request by vikas199**
### Jira Story

- [Snapshot Search duplicate 1-10 results](https://osi-cwds.atlassian.net/browse/SNAP-760)

## Description
When showing more results was clicked for the second time the auto-completer menu was closed suddenly. The onBlur event was the function that was causing the auto-complete menu to close when clicked the second time. As we are using onBlur to close the menu when clicked anywhere on the screen. Had a conversation with Jeff and Tyler as they are ok to remove the onBlur functionality for now.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by vikas199** [comment]
jenkins retest this please

File: app/javascript/common/search/Autocompleter.jsx [hunk 1]
```diff
@@ -205,7 +205,7 @@ export default class Autocompleter extends Component {
       <Autocomplete
         ref={(el) => (this.element_ref = el)}
         getItemValue={(_) => searchTerm}
-        inputProps={{id, onBlur: this.hideMenu, onFocus: this.onFocus}}
+        inputProps={{id, onFocus: this.onFocus}}
```
Comment (wmitchell-cambria) [inline app/javascript/common/search/Autocompleter.jsx]: Hmm, won't this cause a bigger issue of the menu never closing?

Comment (vikas199) [reply app/javascript/common/search/Autocompleter.jsx]: Had a thought on it but again asked Jeff he was ok with this kind of behavior

Comment (wmitchell-cambria) [reply app/javascript/common/search/Autocompleter.jsx]: All right, if it's fine with PO, the code itself is safe enough.

**Comment by vikas199** [comment]
jenkins retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by AVandana-OSI** [comment]
retest this please

**Pull request merged**